website/pages/deployment/setup_tes.mdx

```diff
@@ -0,0 +1,113 @@
+import { Steps, Callout } from "nextra/components"
+
+
+## Funnel TES
+
+[Funnel](https://github.com/ohsu-comp-bio/funnel) is an implementation of the GA4GH Task Execution Schemas. It provides an API server, 
+multiple storage backends, multiple compute backends and a web UI.
+For the purposes of the All In One stack we will be configuring the TRE Agent to use Funnel as the TES compatible execution backend and any outputs from the execution will be stored in the TRE Layer MinIO instance.
+<Steps>
+
+### Set up MinIO Access key
+Navigate to the TRE layer MinIO instance, hosted at `http://localhost:9002` and create access key credentials.
+To login to MinIO use:
+```
+Username: minio
+Password: minio123
+```
+<Callout> 
+Note the name of the TRE output bucket of the Project that you want to submit to. The naming pattern is: ` <projectName> + <uniqueID> + outputtre`. You will need to reference this in the `outputs` section in the [TES message](#submit-a-tes-message)
+
+</Callout>
+
+### Install funnel
+Instructions on how to install Funnel can be found [here](https://ohsu-comp-bio.github.io/funnel/download/)
+
+<Callout> It is currently only available on linux and macOS.</Callout>
+
+### Funnel config file
+Funnel will need access to the TRE layer MinIO. 
+Create a yaml file, point it to the instance of MinIO in the TRE Layer and set the access key & secret created in the previous step.
```

Comment:
I can't see where the access key and secret are created. Could you provide a link to that step?